chain/network/src/types.rs

```diff
@@ -267,23 +247,9 @@ pub enum NetworkRequests {
         target: AccountOrPeerIdOrHash,
     },
     /// Response to state request.
-    StateResponse {
-        route_back: CryptoHash,
-        response: StateResponseInfo,
-    },
-    EpochSyncRequest {
-        peer_id: PeerId,
-        epoch_id: EpochId,
-    },
-    EpochSyncFinalizationRequest {
-        peer_id: PeerId,
-        epoch_id: EpochId,
-    },
```

Comment:
Eg, here, killing epoch sync request could have been submitted as a separate PR which woudl be trivially approved. It's much faster to review a small non-trivial change, and a large trivail change, than a single diff which combines the two, requiring the user to separate "ok, so here we just s/foo/bar" from "ok, so here we are doing something special".